### ISSUE TITLE
[a11y] zlink stats - unlabeld images

### ISSUE THREAD

**Issue by jxjj** [UMN-LATIS/z#236]
Very large number of unlabeled images in infographics, makes it hard to navigate with a screen reader (they should be hidden to screen readers by having empty alt text or aria role = presentation)

audit issue 40
